ncrypt/smime.c

```diff
@@ -2480,6 +2481,9 @@ int smime_class_send_menu(struct Email *e)
         e->security |= SEC_ENCRYPT;
         do
         {
+
+          struct Buffer errmsg = { 0 };
```

Comment:
Please can you change this to `= mutt_buffer_make(0)`
This is our preferred way to create `Buffer`s on the stack.
(It's not widespread in the code, yet, because it's fairly new).

Reply:
done